engine/src/main/java/org/terasology/world/viewer/layers/NominalFacetLayer.java

```diff
@@ -56,8 +56,10 @@ public void render(BufferedImage img, Region region) {
         for (int z = 0; z < height; z++) {
             for (int x = 0; x < width; x++) {
                 Color src = getColor(facet, x, z);
-                int blend = blender.get(src.rgba());
-                dataBuffer.setElem(z * width + x, blend);
+                if (src != null) {
```

Comment:
Actually, my idea here was that `src` should never be `null`. Did you find a scenario where this is not the case?